### ISSUE TITLE
Revise the architecture graphic

### ISSUE THREAD

**Issue by stichbury** [kedro-org/kedro#2915]
The graphic on the architecture documentation page (currently on the wiki https://github.com/kedro-org/kedro/wiki/Architecture#kedro-architecture-overview but moving back into docs soon) needs some small updates, as pointed out by @noklam 

* Cli.py, it still work but we don't ship it to starters anymore
* Mini-kedro should be renamed to standalone catalog?

Just got to dig out the source and fix it...hopefully...rather than redraw.

**Comment by yetudada** [comment]
I'd also add that we should improve the overall accessibility of this diagram so that it's easy for all people to understand what they're looking at.

**Comment by stichbury** [comment]
Agreed, @yetudada. Next step is a meeting with engineering to pair on what each element in the diagram does and I'll record it and make some copy for a blog post or on the page itself. So this ticket should stay open for now.
